website/docs/docs/dbt-cloud-apis/metadata-use-case-guides.md

```diff
@@ -0,0 +1,1143 @@
+---
+title: "Use cases"
```

Comment:
renamed this to 'using' instead of 'use cases' per guidance from Leona. does this read better as 'use cases' or 'using the metadata api'?

Reply:
Either is fine! We'll want to make it clear that these are just a few simple examples though, not exhaustive of everything one can do with the API